content/blog/2022/2022-08-25-schedule-cron-job-in-node.md

```diff
@@ -0,0 +1,277 @@
+
+
+### Node cron syntax
+```
+cron.schedule(expression , function, options);
+```
+- **The first argument** passed to the node-cron syntax is a cron-expression (crontab). This expression is used to specify when a cron job is to be executed.
+
+    The cron expression used here can be represented using Asterix  (*),  which are wildcard characters representing the scheduled time for cron jobs. The expression by default will be in the format `* * * * * *`.
+
+Here's a quick reference to the cron expression, with asterisks indicating what each one represents.
+
+```
+ # ┌────────────── second (0 - 59) (optional)
+ # │ ┌──────────── minute (0 - 59) 
+ # │ │ ┌────────── hour (0 - 23)
+ # │ │ │ ┌──────── day of the month (1 - 31)
+ # │ │ │ │ ┌────── month (1 - 12)
+ # │ │ │ │ │ ┌──── day of the week (0 - 6) (0 and 7 both represent Sunday)
+ # │ │ │ │ │ │
+ # │ │ │ │ │ │
+ # *  * * * * *  command to execute
+```
+You can replace each asterisk with an appropriate number (or character) such that the expression describe the time the job will be executed.
+
+The first field (second) is optional. Each field can have one or more values separated by commas, or a range of values separated by hyphens.
+
+- `*` Asterisk operator this is a wildcard and acts as a placeholder. It means a value or always. Eg. If the asterisk symbol is in the month field, it means performing these specified tasks every month.
+- `,` The comma operator allows you to specify a list of values for repetition. For example, If you have `1, 3, 5` in the `month field`, the task will run on days of `month 1, 3, and 5`.
+- `-` The hyphen operator allows you to specify a range of values. If you have 1-5 in the `Day of the week field`, the task will run every weekday (From Monday to Friday).
+- `/` The slash operator allows you to specify values that will be repeated over a certain interval between them. For example, if you have */4 in the Hour field, It means the action will be performed every 4 hours. 
+
+A single asterisk means the specified job will run for every instance of that unit of time it represents. 
+
+Eg. cron expressions are started with five asterisks `* * * * *` by default this means: 
+
+Perform the specified task `at every minute`, `of every hour`, `of every day`, `of every month`, `and of every day of the week`.
+
+If you're unsure about manually writing cron syntax, you can, use free tools like [Crontab Generator](https://crontab-generator.org/) or [Crontab.guru](https://crontab.guru/#*_*_*_*_*) to generate the exact time date you want for cron expression.
+
+- **The second argument** specifies a function. This is the task that executes at intervals.  When the time specified in the first argument is reached, this function is called.
+
+- **The third argument** is a configuration object for the scheduled jobs, which is optional.
+
+#### The available options are:
```

Comment:
This shouldn't be a section in itself. Only use section headers if it would make sense to see this in the table of contents (this doesn't show up in the TOC, but only because the TOC only shows the first two levels of headers).